src/analyzer/analyzerrhythm.cpp

```diff
@@ -450,7 +460,18 @@ void AnalyzerRhythm::computeMeter() {
             bestHierarchy = meter;
         }
     }
-    qDebug() << bestHierarchy;
+    auto pulseValues = accumulatedPulsesLenghtsAndWeights.keyValueBegin();
```

Comment:
Do we want to use `auto` here? I have no idea what type `pulseValues` is from looking at the right hand side of the assignment.